lib/tasks_private/spec_helper.rake

```diff
@@ -4,7 +4,11 @@ namespace :spec do
     tower_host = ENV['TOWER_URL'] || "https://dev-ansible-tower3.example.com/api/v1/"
     id = ENV['TOWER_USER'] || 'testuser'
     password = ENV['TOWER_PASSWORD'] || 'secret'
-    PopulateTower.new(tower_host, id, password).create_dataset.counts
+    populate = PopulateTower.new(tower_host, id, password)
+
+    populate.create_dataset
+    populate.counts
+    populate.to_file(ManageIQ::Providers::AnsibleTower::Engine.root.join("spec/support/tower_data.yml"))
```

Comment:
single `'` 😄

Reply:
Should I open a new PR? :smile:

Reply:
I don’t like the fact that the path is here in the Rake task. Since the YAML is going to be used by the Specs, I’d prefer to have the constant somewhere along them and use it from here.